packages/acs-ui-common/package.json

```
@@ -7,7 +7,7 @@
   "types": "dist/acs-ui-common.d.ts",
   "scripts": {
     "build": "rushx build:esm && rushx build:cjs && api-extractor run --local",
```

Should we just be cleaning the whole dist folder here instead of just the dist-esm folder?

```suggestion
    "build": "rushx clean && rushx build:esm && rushx build:cjs && api-extractor run --local",
    "clean": "rimraf dist"
```

---

Done